src/locales/de/translations.json

```diff
@@ -160,5 +160,8 @@
       "link": "Karriere"
     }
   },
+  "image": {
+    "attribution": "Foto von "
+  },
```

Comment:
Ideally you should provide an interpolation for things like that as you can't know the sentence structure of the future translation. See https://www.i18next.com/translation-function/interpolation. Something like `"key": "{{what}} is {{how}}`.

For here it's okay and makes things easier as we have a component involved anyway.

Reply:
I will remember that for next time 👍